src/screens/ChangePassword/index.js

```diff
@@ -0,0 +1,163 @@
+import React, { useContext, useState, useEffect } from 'react';
+import {
+  View,
+  Text,
+  TextInput,
+  StyleSheet,
+  TouchableOpacity,
+  Button,
+} from 'react-native';
+import Modal from 'react-native-modal';
+import { Context as AuthContext } from '../../state/AuthContext';
+
+export default function SignUpScreen({ navigation }) {
```

Comment:
why is this component called SignUpScreen? shouldn't it be ChangePassword?

Reply:
or is it because it's inside Signupscreen ?

Reply:
true, i will fix them both